### DISCUSSION TITLE
Ability to ask out a specified time period?

### DISCUSSION THREAD

**Discussion by samuell** [featureform/featureform#444] [Q&A]
Not sure if this is a Q&A or an idea, but here goes ...

Is it possible to query out a set of features for a specific time period? If not, that is something that would be awesome.

The use case we are thinking of is where we have years of time series data and want to get a dataset for say the last month, or something similar that is easy enough for a data scientist to play around with interactively, visualize with some plotting etc.

**Comment by mallman** [comment]
Have you tried this? https://docs.featureform.com/concepts/dataframes
